src/System.Runtime.CompilerServices.Unsafe/tests/UnsafeTests.cs

```diff
@@ -459,6 +459,24 @@ public static void RefAdd()
         }
 
         [Fact]
+        public static unsafe void VoidPointerAdd()
+        {
+            int[] a = new int[] { 0x123, 0x234, 0x345, 0x456 };
+
+            fixed (void* ptr = a)
+            {
+                void* r1 = Unsafe.Add<int>(ptr, 1);
```

Comment:
I don't understand what you mean exactly. Can you give me an example test case?
Since this is an Unsafe function call, there are no bounds checks to prevent buffer overruns.

Is something like this what you had in mind? If so, I don't know if a check like this is very useful.
```C#
void* underflow = Unsafe.Add<int>(ptr, -1);
Assert.Equal(0, *(int*)underflow);

void* overflow = Unsafe.Add<int>(ptr, 4);
Assert.Equal(0, *(int*)overflow);
```